docs/developers/setup.md

```diff
@@ -0,0 +1,132 @@
+# Development environment
+
+<!-- markdownlint-disable MD046 -->
+
+This section outlines some suggestions as well as conventions used by the EMMOntoPy developers, which should be considered or followed if one wants to contribute to the package.
+
+## Setup
+
+!!! important "Requirements"
+    This section expects you to be running on a Unix-like system (e.g., Linux) with *minimum* Python 3.6.
+
+### Virtual environment
+
+Since development can be messy, it is good to separate the development environment from the rest of your system's environment.
+
+To do this, you can use a virtual environment.
+There are a several different ways to create a virtual environment, but we recommend using either `conda` or `virtualenv`.
```

Comment:
I think it is fine to come with such a recommendation, but does it mean that maind evelopers should do one of these? (I usually make new environments with python -m venv ... ).

Reply:
This is an excellent point!
A quick search got me [this brilliant answer on StackOverflow](https://stackoverflow.com/a/41573588/12404091).

Personally, I use `virtualenv` and `virtualenvwrapper`.

I think I will add the link to the StackOverflow answer in the documentation here, what do you think?
If you'd like to add your comments on the use of `venv`, I think that would be great.
My main issue with this section in the documentation is not so much `venv` versus `virtualenv`, but rather `conda`, since I don't know much about it and wouldn't be able to troubleshoot it in any way.

Reply:
That link is good, although can also be a bit confusing. It also shows that much of this boils down to personal preferences and not one 'best' answer. It can be added as an example.